keyboards/kbd75/keymaps/chester_web/keymap.c

```diff
@@ -0,0 +1,229 @@
+#include "kbd75.h"
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```

Comment:
This is actually defined in the core as `MOD_MASK_CTRL`, and it doesn't look like you're actually using it.

```suggestion
```